frontend/amundsen_application/static/js/features/ColumnList/index.tsx

```diff
@@ -206,7 +209,7 @@ const ExpandedRowComponent: React.FC<ExpandedRowProps> = (
           editUrl={rowValue.editUrl || undefined}
         >
           <ColumnDescEditableText
-            columnIndex={rowValue.index}
+            columnIndex={rowValue.sort_order}
```

Comment:
Personally not a fan of referring to columns by index, especially since we introduced sorting. Should eventually use a column ID of sorts.